src/components/Connect/Burner/ImportPk/index.tsx

```diff
@@ -63,24 +58,33 @@ export default function ImportSeed({
               label='Please enter your private key below to continue'
               vertical={true}
               centered={true}
-              autofocus={true}
+              autoFocus={true}
               value={privateKey}
               importError={importError}
               onChange={setPrivateKey}
               setIsValid={setIsValid}
             />
           )}
-
-          <ButtonRowSpaced>
-            <ButtonV2 large disabled={loading} onClick={back} type='button'>
-              Back
-            </ButtonV2>
-            <ButtonV2 large disabled={loading} green type='submit'>
-              Connect
-            </ButtonV2>
-          </ButtonRowSpaced>
-        </form>
-      </ShadowBox>
+        </ShadowBox>
+        <ButtonRowSpaced>
+          <ButtonV2 large disabled={loading} onClick={back} type='button'>
+            Back
+          </ButtonV2>
+          <ButtonV2 large disabled={loading} green type='submit'>
+            Connect
+          </ButtonV2>
+        </ButtonRowSpaced>
+      </form>
```

Comment:
I think it's nicer to have the buttons below the `ShadowBox` like in other similar screens. We could opt for buttons inside the box, but then I think we should do that in other places as well